README.md

```diff
@@ -41,19 +46,24 @@ terraform apply
 * Run `docker-run.sh -h` to see the help menu
 
 ```
-$ ./docker-chef.sh -h
+[user@host ~]$ ./docker-chef.sh -h
 This is a control script for starting|stopping Chef Server and Chef Automate docker services.
 
 You must specify the following options:
  -s [automate|chef-server]           REQUIRED: Services type: Chef Server or Chef Automate
  -a [stop|start]                     REQUIRED: Action type: start or stop services
  -n [container name]                 OPTIONAL: The docker container name. Leaving blank implies ALL
+ -l [path]                           OPTIONAL: Apply the Automate License from [path]
```

Comment:
What happens if you combine `-l` and an action ( `-a start` or `stop`)?   I don't see any script logic that makes those mutually exclusive, but I suspect it won't go so well if those are combined?

Reply:
Addressing.